### PR TITLE
Prepping v6.1.0 release

### PR DESCRIPTION
**Pull request by jessebeach**
- CHANGELOG updates for upcoming version 6.1.0

### CODE REVIEW

**Comment by ljharb** [comment]
oof, i didn't realize the eslint 5 stuff would require a semver-major :-/

could we maybe backport that into a v6 minor?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+7.0.0 / 2018-06-25
+==================
+- [new] Support for eslint v5, #451
```
Comment (ljharb) [inline CHANGELOG.md]: i don't see any breaking changes indicated here, "new" means minor?

**Comment by jessebeach** [comment]
@ljharb @evcohen I would like to deprecate `label-has-for` in this release. We'll leave it in the plugin, but add `-deprecated` to the rule name and indicate that it will disappear in a future release. `label-has-for` is superseded by `label-has-associated-control`.

File: CHANGELOG.md (hunk 1 above)
Comment (ljharb) [reply CHANGELOG.md]: i definitely do think so :-) `v6.1.0` seems ideal here (even if you also want to add `deprecated: true` to `label-has-for`)

**Pull request merged**